workflows/epigenetics/atacseq/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.10] 2023-11-10
+
+### Automatic update
+- `toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/2.18.2.4` was updated to `toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicatesWithMateCigar/2.18.2.3`
```

Comment:
I think planemo autoupdate does substring matching on `picard_MarkDuplicates`, that is probably a bug

Reply:
Yes it is always the same bug I did not have time to report and fix...